.travis.yml

```diff
@@ -68,13 +68,16 @@ before_install:
     - python -c "import fcntl; fcntl.fcntl(1, fcntl.F_SETFL, 0)"
     - CONTAINER="wheels";  # pre-relesae
     - BUILD_COMMIT=$BUILD_COMMIT;
-    - BUILD_DEPENDS="$NP_BUILD_DEP Cython==0.28.2"
+    # add setuptools_scm here as WAR for https://github.com/pypa/packaging-problems/issues/134 with Python 3.5
+    - BUILD_DEPENDS="$NP_BUILD_DEP Cython==0.28.2 setuptools_scm"
```

Comment:
Should setuptools_scm and the pip (re?)install only be done for the 3.5 build?

Reply:
is it relevant that pandas requires cython 0.29.13?

Reply:
This is on the `master`, which sets the `BUILD_COMMIT=v0.25.1` which allowed older Cythons.

Reply:
yeah @TomAugspurger its only needed for the mac 3.5 build. I've added some more explanation in https://github.com/pandas-dev/pandas/issues/28954#issuecomment-541896439.